src/python/WMCore/Storage/Backends/XRDCPImpl.py

```diff
@@ -115,7 +117,14 @@ def createStageOutCommand(self, sourcePFN, targetPFN, options=None, checksums=No
         if args.wma_disablewriterecovery:
             copyCommand += "env XRD_WRITERECOVERY=0 "
 
-        copyCommand += "%s --force --nopbar " % xrdcpExec
+        if args.wma_preload:
+            xrdcpCmd = "%s %s" % (args.wma_preload, xrdcpExec)
+            self.xrdfsCmd = "%s xrdfs" % args.wma_preload
+        else:
+            xrdcpCmd = xrdcpExec
+            self.xrdfsCmd = "xrdfs"
```

Comment:
This line isn't really needed.

Reply:
It's redundant, yes. Makes the code a bit easier to read and understand IMO.

Reply:
Still it's an unneeded code duplication. Anyways, not a big deal.